### PR TITLE
Reformat build.xml files using tidy.

### PR DESCRIPTION
**Pull request by PeterJohnson**
The tidy usage is documented in the xml-tidy.conf configuration file.

### CODE REVIEW

**Comment by AustinShalit** [comment]
Do the Examples XML files need to be formatted also?

**Comment by PeterJohnson** [comment]
No, they look okay.  They also use a different amount of indentation (4 instead of 2).

File: edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml [hunk 1]
```diff
@@ -1,264 +1,293 @@
...
+      <pathelement location="${wpilib.ant.dir}/commons-logging-1.2.jar" />
     </classpath>
   </typedef>
-
-  <target name="clean" description="Clean up all build and distribution artifacts.">
-    <delete dir="${build.dir}"/>
-    <delete dir="${dist.dir}"/>
+  <target name="clean"
+          description="Clean up all build and distribution artifacts.">
+
```
Comment (AustinShalit) [inline edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml:36]: Should this whitespace be here?

Comment (PeterJohnson) [reply edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml:36]: No, but I don't know if I care.  I much prefer automatically formatted even if it has a few inconsistencies.

**Review by AustinShalit — APPROVED** [review]

**Pull request merged**